deploy/olm-catalog/kubevirt-ssp-operator/0.16.0/kubevirt-ssp-operator.v0.16.0.clusterserviceversion.yaml

```diff
@@ -1,8 +1,10 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
-  creationTimestamp: null
-  name: kubevirt-ssp-operator.v0.1.1
+  annotations:
+    alm-examples: '[{"apiVersion":"kubevirt.io/v1","kind":"KubevirtCommonTemplatesBundle","metadata":{"name":"kubevirt-common-template-bundle"},"spec":{"version":"v0.4.1"}},{"apiVersion":"kubevirt.io/v1","kind":"KubevirtTemplateValidator","metadata":{"name":"kubevirt-template-validator"},"spec":{"version":"v0.1.1"}}]'
```

Comment:
You can also spread this over multiple lines to improve readability, an example is in [kubevirt's csv template](https://github.com/kubevirt/kubevirt/blob/master/manifests/release/olm/bundle/kubevirtoperator.VERSION.csv.yaml.in#L9-L22)

Reply:
I'd like to, but OTOH this annotation is autogenerated, so I'd like to automate the change. I'll postpone this due to time constraints.